core/wiki/macros/dragndrop.tid

```diff
@@ -0,0 +1,378 @@
+\end
+
+\procedure draggable-droppable-top-drop-actions()
+<$action-setfield
+	$tiddler={{{ [<dragStateTiddler>addsuffix[/drag-handled]] }}}
+	text="yes"
+/>
+<%if [<actionTiddler>removeprefix<prefix>removesuffix<suffix>] %>
+	<$let
+		actionTiddler={{{ [<actionTiddler>removeprefix<prefix>removesuffix<suffix>] }}}
+	>
+		<$transclude
+			$variable="draggable-droppable-top-drop-actions-inner"
+		/>
+	</$let>
+<% else %>
+	<$transclude
+		$variable="draggable-droppable-top-drop-actions-inner"
+	/>
+<% endif %>
+\end
+
+\procedure draggable-droppable-drag-end-actions()
+<%if [<dragStateTiddler>addsuffix[/drag-handled]is[missing]] %>
+	<$let
+		fromList={{{ [<dragStateTiddler>get[start-from-list]] }}}
+		fromIndex={{{ [<dragStateTiddler>get[start-from-index]] }}}
+		listBeforeTiddler={{{ [list<fromList>zth<fromIndex>] }}}
+		actionTiddler={{{ [<actionTiddler>removeprefix<prefix>removesuffix<suffix>] :else[<actionTiddler>] }}}
+	>
+		<%if [<listBeforeTiddler>!is[blank]] %>
+			<$action-listops
+				$tiddler=<<fromStoryList>>
+				$subfilter="+[insertbefore<actionTiddler>,<listBeforeTiddler>]"
+			/>
+		<% else %>
+			<$action-listops
+				$tiddler=<<fromStoryList>>
+				$subfilter="[<actionTiddler>]"
+			/>
+		<% endif %>
+		<$action-deletetiddler
+			$tiddler={{{ [<dragStateTiddler>addsuffix[/drag-handled]] }}}
+		/>
+		<$action-deletetiddler
+			$tiddler=<<dragStateTiddler>>
+		/>
+	</$let>
+<% else %>
+	<$action-deletetiddler
+		$tiddler={{{ [<dragStateTiddler>addsuffix[/drag-handled]] }}}
+	/>
+<% endif %>
+\end
+
+\procedure draggable-droppable-drag-start-actions()
+<$let
+	nth={{{ [list<list>allbefore<tiddler>count[]] }}}
+	listLength={{{ [list<list>count[]subtract[1]] }}}
+>
+	<%if [<nth>match<listLength>] %>
+		<$action-setfield
+			$tiddler=<<dragStateTiddler>>
+			before-tiddler={{{ [list<list>before<tiddler>] }}}
+		/>
+	<% endif %>
+	<$action-setfield
+		$tiddler=<<dragStateTiddler>>
+		drag-count="0"
+		nth=<<nth>>
+		height=<<tv-selectednode-height>>
+		width=<<tv-selectednode-width>>
+		margin-bottom=<<styleMarginBottom>>
+		start-next-tiddler={{{ [list<list>after<tiddler>] }}}
+		start-from-index={{{ [list<list>allbefore<tiddler>count[]] }}}
+		start-from-list=<<list>>
+	/>
+	<$action-setfield
+		$tiddler=<<dragStateTiddler>>
+		drag-tiddler=<<tiddler>>
+		next-tiddler={{{ [list<list>after<tiddler>] }}}
+		from-index={{{ [list<list>allbefore<tiddler>count[]] }}}
+		from-list=<<list>>
+		from-history=<<history>>
+		target={{{ [list<list>after<tiddler>] }}}
+	/>
+	<$action-setfield
+		$tiddler=<<dragStateTiddler>>
+		tags="$:/tags/DragStateTiddler"
+		text="yes"
+	/>
+	<%if [<modifier>!match[ctrl]] %>
+		<$action-listops
+			$tiddler=<<list>>
+			$subfilter="-[<tiddler>]"
+		/>
+	<% else %>
+		<$action-setfield
+			$tiddler=<<dragStateTiddler>>
+			copy="yes"
+		/>
+	<% endif %>
+</$let>
+\end
+
+\function get.drag.state.tiddler.height() [<dragStateTiddler>get[height]addsuffix[px]]
```

Comment:
There is already a convention for core global filter naming. They should be prefixed with `tf.` and named-with-hyphens.

In this case, all functions use a `get.` prefix, which imo is redundant, since functions always "get" a variable

So I thin for consistency reasons it should be named `tf.drag-state-tiddler-height`